src/System.Drawing.Common/tests/mono/System.Drawing.Drawing2D/GraphicsPathIteratorTest.cs

```diff
@@ -0,0 +1,170 @@
+//
```

Comment:
We need to include a 2-line header in all of these new files.

```C#
// Licensed to the .NET Foundation under one or more agreements.
// See the LICENSE file in the project root for more information.
```

Reply:
Just to be clear, do you need this header in addition to the existing header or does it replace the existing header?

Reply:
It is in addition to the existing header -- it just goes at the very top.

Reply:
The files under System.Drawing.Common/src/Unix show what it should look like. Those are files pulled in from the mono codebase.